src/resources/schema.ts

```diff
@@ -72,4 +72,14 @@ export const resourceSchema = new mongoose.Schema({
   sequence: Boolean,
   status: String,
   relations: [{type: relationSchema, required: false}],
-}, {timestamps: {createdAt: 'dateAdded', updatedAt: 'dateModified'}});
\ No newline at end of file
+}, {timestamps: {createdAt: 'dateAdded', updatedAt: 'dateModified'}});
+
+resourceSchema.pre('save', function(this: any, next: any) {
+  if (this.languages && this.languages.iso639_3) {
+    // filter out empty strings
+    this.languages.iso639_3 = this.languages.iso639_3.map((lang: string) => lang || "zxx")
```

Comment:
@gliese1337 Do you know what the resource library's behavior could have been in the case that the ISO lang code was an empty string? It's breaking the configuration modal because it's not a valid lang code.

Reply:
I do not know for certain, but returning a Bad Request error code or replacing it with "zxx" seem like reasonable options, If there are other valid language codes, just ignoring empty strings and falling back to "zxx" when the list is empty would probably be the most friendly behavior.